controllers/suite_test.go

```diff
@@ -91,10 +94,15 @@ var _ = BeforeSuite(func() {
 	Expect(k8sClient).ToNot(BeNil())
 
 	// Kubernetes Manager
-	syncPeriod := 30 * time.Second
 	k8sManager, err := ctrl.NewManager(cfg, ctrl.Options{
 		Scheme:     scheme.Scheme,
 		SyncPeriod: &syncPeriod,
+		Controller: v1alpha1.ControllerConfigurationSpec{
+			GroupKindConcurrency: map[string]int{
+				"Workspace.app.terraform.io": 5,
+				"Module.app.terraform.io":    5,
+			},
+		},
```

Comment:
This is not really related to this issue, but this will be useful for further improvements to make tests parallel. 🙃